tests/console/login.go

```diff
@@ -141,9 +141,10 @@ func LoginToFedora(vmi *v1.VirtualMachineInstance) error {
 	}
 
 	// Do not login, if we already logged in
+	loggedInPromptRegex := fmt.Sprintf(`\[(fedora|root)@(localhost|fedora|%s) (~|fedora)\](\$|\#) `, vmi.Name)
```

Comment:
The previous expression was more specific.
Here you accept for example `root@fedora ~$` which is wrong.

Reply:
Why is it wrong and why do we care? The format is correct.

Reply:
Personally I do not mind reverting to the previous regex, this was a request from the reviewers and a good example why it is not a good thing to do many things in one PR.

Reply:
as long as it doesn't enter the wrong case of the expecter (i.e the login one)
we don't need to be 100% accurate imo